src/components/Main.tsx

```diff
@@ -734,56 +729,124 @@ const EdgeAppStack = () => {
 }
 
 const EdgeTabs = () => {
-  const dispatch = useDispatch()
-
   return (
     <Tab.Navigator
-      initialRouteName="walletList"
+      initialRouteName="walletsTab"
       tabBar={props => <MenuTabs {...props} />}
       screenOptions={{
-        title: '',
-        headerTitle: ({ children }: HeaderTitleProps) => <HeaderTitle title={children} />,
-        headerLeft: () => <HeaderTextButton type="help" placement="left" />,
-        headerRight: () => <SideMenuButton />,
-        headerShown: true,
-        headerTransparent: true,
+        headerShown: false,
         unmountOnBlur: true
       }}
     >
-      <Tab.Screen
-        name="walletList"
-        component={WalletListScene}
+      <Tab.Screen name="walletsTab" component={EdgeWalletsTabScreen} />
+      <Tab.Screen name="buyTab" component={EdgeBuyTabScreen} />
+      <Tab.Screen name="sellTab" component={EdgeSellTabScreen} />
+      <Tab.Screen name="exchangeTab" component={EdgeExchangeTabScreen} />
+      <Tab.Screen name="marketsTab" component={EdgeMarketsTabScreen} />
+      <Tab.Screen name="extraTab" component={ExtraTabTabScene} />
+    </Tab.Navigator>
+  )
+}
+
+const EdgeWalletsTabScreen = () => {
+  return (
+    <Stack.Navigator initialRouteName="walletList" screenOptions={defaultScreenOptions}>
+      <Stack.Screen name="walletList" component={WalletListScene} options={firstSceneScreenOptions} />
+      <Stack.Screen
+        name="transactionList"
+        component={TransactionList}
+        listeners={{
+          focus: () => {
+            requestPermission('contacts').catch(showError)
+          }
+        }}
+      />
+      <Stack.Screen
+        name="transactionDetails"
+        component={TransactionDetailsScene}
         options={{
-          headerTitle: EdgeLogoHeader
+          headerTitle: () => <TransactionDetailsTitle />
+        }}
+        listeners={{
+          focus: () => {
+            requestPermission('contacts').catch(showError)
+          }
         }}
       />
-      <Tab.Screen
+    </Stack.Navigator>
+  )
+}
+
+const EdgeBuyTabScreen = () => {
+  return (
+    <Stack.Navigator initialRouteName="pluginListBuy" screenOptions={defaultScreenOptions}>
+      <Stack.Screen
         name="pluginListBuy"
         // @ts-expect-error
-        component={GuiPluginListScene}
-        route={{ params: { direction: 'buy' } }}
+        component={GuiPluginListBuyScene}
+        options={firstSceneScreenOptions}
```

Comment:
Should we put copies of GuiPluginViewScene in here?

Reply:
No, because the tabs don't remember state.

Reply:
Though it may be useful to be able to switch out of the plugin view scene to the wallet scene with one tap vs a bunch of back taps to get out of this scene...